metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -768,6 +787,7 @@ def event_processor(
             if (
                 event.timestamp < self.config.start_time
                 or event.timestamp >= self.config.end_time
+                or not self._is_table_allowed(event.resource)
             ):
```

Comment:
Any need for logging/reporting when this happens?

Reply:
This is normal filtering similar to any other source. I don't think anything is required here.